subprojects/docs/src/docs/release/notes.md

```diff
@@ -34,22 +34,34 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 <a name="cache"></a>
 ## Gradle's Dependency Cache can be relocated
 
-The Gradle Dependency cache under `$GRADLE_HOME/caches/modules-2` can now be relocated to another directory or host for dependencies cached by Gradle 6.1 and later. When moved to a new location or seeded into a host image, builds using the dependency cache will not need to access the network to download artifacts or metadata if the dependencies have already been downloaded. 
+It has become a common practice to run builds in a CI/CD pipeline with ephemeral agents, meaning single use environments.
+This practice is most often used to improve reliability and reduce potential interactions between builds.
+It however results in longer build times as nothing can be reused from build to build.
+
+A significant overhead caused by this practice with Gradle is the dependency downloads.
+Every build has a longer duration caused by the fetch of all dependencies.
+ 
+With Gradle 6.1, [the dependency cache](userguide/dependency_resolution.html#sec:dependency_cache) can be relocated, to another directory or host, for dependencies cached by Gradle 6.1 and later.
+When moved to a new location or seeded into a host image, builds will be able to leverage the dependency cache and will not need to access the network to download artifacts or metadata if they are already cached. 
 
 Note that the cache should be primed and consumed by same Gradle version for maximum effect.
 See [the documentation](userguide/dependency_resolution.html#sub:cache_copy) for more details. 
 
-This is just one step along the way to help organizations using ephemeral CI agents to reduce the overhead of downloading dependencies during the build.
-
 <a name="compilation-order"></a>
-## Defining compilation order between Groovy, Scala and Java
+## Defining compilation order between JVM languages
 
-Previously, the relationship between Java, Groovy and Scala compilation was hardcoded with explicit task dependencies in the same project.
-Gradle assumed Groovy and Scala compilation would always depend on Java compilation. That is, `compileGroovy` and `compileScala` would directly depend on the output from `compileJava`.
+Gradle supports combining multiple JVM language in a single project, like mixing Scala and Java or Java, Groovy and Kotlin for example.
+Each language will have its own source set and compile task defined.
+Prior to Gradle 6.1, the compilation dependencies were defined in a way that made them hard to tweak.
```

Comment:
`compilation` -> `compile task` (?)

Otherwise it sounds very much like "compile dependency" (in the dependency management sense with implementation/compile configuration).